app/Http/Requests/TaskComment/UpdateTaskCommentRequest.php

```diff
@@ -16,6 +16,9 @@ public function _authorize(): bool
 
     public function _rules(): array
     {
-        return [];
+        return [
```

Comment:
without this rule, the ItemController does not find the id key

Reply:
Because you have changed it at https://github.com/cattr-app/backend-application/pull/45/files/23ec22f04a37fe857d4484bcb9fc6441a524876a#diff-fc8205c776dae5f3ecd772e0b66cfdebbb0c50780abab479bd4a47d8bd711044R161

Reply:
![image](https://user-images.githubusercontent.com/51990904/227733670-67b43ade-7e04-4833-ade5-dc2d05825498.png)
![image](https://user-images.githubusercontent.com/51990904/227733699-0d7e5902-3c48-48d5-b98d-fe935b88d862.png)
![image](https://user-images.githubusercontent.com/51990904/227733760-b23d3a95-b232-4cb9-89ad-1a2e6d2bbad2.png)
![image](https://user-images.githubusercontent.com/51990904/227733776-251775bf-05ef-4b47-b218-9cd2853bd943.png)
for example 
![image](https://user-images.githubusercontent.com/51990904/227733799-d5ddc7c5-c41d-45d4-945c-629239ad156a.png)
without this params in rules logic edit commit do not work
![image](https://user-images.githubusercontent.com/51990904/227733863-b6a9e560-dc85-4335-a747-1431154cbef5.png)